typo3/cms-core/2020-11-17-4.yaml

```diff
@@ -0,0 +1,7 @@
+title: 'TYPO3-CORE-SA-2020-012: XML External Entity in Dashboard Widget'
+link: 'https://typo3.org/security/advisory/typo3-core-sa-2020-012/'
+branches:
+    10.x:
+        time: '2020-11-17 08:51:21'
+        versions: ['>=10.0.0', '<10.4.10']
+reference: 'composer://typo3/cms-core'
```

Comment:
should be assigned to `typo3/cms-dashboard`

Reply:
None of the TYPO3 packages can be used without `typo3/cms-core`, that's the reason we're using it for these advisories. Each of the packages have the same versions since they are maintained in a monolithic repository and subtree-split into those packages. Basically we did it like that during the last few years...

Since none of these `typo3/cms-*` packages can be used standalone, I don't see a reason for making things more complicated here. I'm giving it up for the time being...

Reply:
@ohader but is it enforced that you have the exact same version of `typo3/cms-core` and `typo3/cms-fluid` ?

IIRC, the generation of these advisories has been scripted by the typo3 team based on the advisory pages. And these pages do contain that info already (see `ext: core` vs `ext: dashboard` in the `Subcomponent` info) so that could be extracted automatically.

Reply:
Okay, let's have a quick look into that:
* https://packagist.org/packages/typo3/cms-fluid#v10.4.10 → require `typo3-core:10.4.10` (exact version, no `^`, no `~`)
* https://packagist.org/packages/typo3/cms-core?query=typo3%2Fcms&type=typo3-cms-framework#v10.4.10 → same for all subtree-split packages `typo3/cms-*`
* all(!) packages require a specific version of `typo3/cms-core`

All of those packages have to be used for the exact same version - as the TYPO3 core monolith is maintained and released as a whole.